packages/cms-base/components/public/cms/element/CmsElementImage.vue

```diff
@@ -74,6 +77,9 @@ const imageComputedContainerAttrs = computed(() => {
       <source :src="imageAttrs.src" :type="mimeType" />
       Your browser does not support the video tag.
     </video>
+    <ClientOnly v-else-if="isSpatial(props.content.data.media)">
+      <SwMedia3D :src="props.content.data.media.url" />
```

Comment:
Let's make this component lazy loaded. Otherwise import from line `12` is always executed in every `CmeElementImage` even if it's a normal picture